api/src/main/kotlin/team/backend/domain/entity/MemberQuizAnswer.kt

```diff
@@ -0,0 +1,38 @@
+package team.backend.domain.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import team.backend.infrastructure.BaseEntity
+
+@Entity
+@Table(name = "tbl_member_quiz_answer")
+class MemberQuizAnswer(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "member_quiz_answer_id")
+    val id: Long? = null,
```

Comment:
all of the fields in the entity have default values.
therefore, there seems to be a lot of boilerplate code in the project.
using the no-arg plugin would likely reduce duplication.

Reply:
Unnecessary default values have been removed.